sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Filtering/CollectionConfiguration.cs

```diff
@@ -38,7 +38,7 @@ internal class CollectionConfiguration
         #endregion
 
         #region Metadata used by other components
-        private readonly List<Tuple<string, DerivedMetricInfoAggregation?>> telemetryMetadata = new List<Tuple<string, DerivedMetricInfoAggregation?>>();
+        private readonly List<Tuple<string, Models.AggregationType?>> telemetryMetadata = new List<Tuple<string, Models.AggregationType?>>();
```

Comment:
Is there a reason to use `Model.*` pattern instead of just `AggregationType`. I see this pattern in may places.

Reply:
This is necessary because there are two classes named `AggregationType`. 
- One of the generated classes, introduced in this PR: `Models.AggregationType` 
https://github.com/Azure/azure-sdk-for-net/blob/3c1312bcbc42670445a5a67771d1451cc281ca20/sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Generated/Models/AggregationType.cs
- and one of the classes from AI SDK: `Filtering.AggregationType`.
https://github.com/Azure/azure-sdk-for-net/blob/3c1312bcbc42670445a5a67771d1451cc281ca20/sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Filtering/AggregationType.cs